Packages/nadena.dev.modular-avatar/Editor/AvatarProcessor.cs

```diff
@@ -184,6 +184,7 @@ public static void ProcessAvatar(GameObject avatarGameObject)
                     new VisibleHeadAccessoryProcessor(vrcAvatarDescriptor).Process(context);
                     new RemapAnimationPass(vrcAvatarDescriptor).Process(context.AnimationDatabase);
                     new BlendshapeSyncAnimationProcessor().OnPreprocessAvatar(avatarGameObject, context);
+                    ObjectRemoverProcessor.OnPreprocessAvatar(avatarGameObject);
```

Comment:
This is a rather late phase of processing to remove objects at - it means that any menu installers, etc will still be applied.

Reply:
moved to the top as this is actually valid use, people could create multiple versions of same avatar by just creating prefab variants and removing certain elements including the modular avatar components